libr/include/r_cons.h

```diff
@@ -503,39 +505,38 @@ typedef struct r_cons_t {
-#define RColor_BGBLUE    RCOLOR(ALPHA_BG,     0x00, 0x00, 0xff, 0x00, 0x00, 0x00)
-#define RColor_GRAY      RCOLOR(ALPHA_FG,     0x7f, 0x7f, 0x7f, 0x00, 0x00, 0x00)
-#define RColor_BGGRAY    RCOLOR(ALPHA_BG,     0x7f, 0x7f, 0x7f, 0x00, 0x00, 0x00)
-#define RColor_BBLACK    RCOLOR(ALPHA_BOLD,   0x00, 0x00, 0x00, 0x00, 0x00, 0x00)
-#define RColor_BRED      RCOLOR(ALPHA_BOLD,   0xff, 0x00, 0x00, 0x00, 0x00, 0x00)
-#define RColor_BGBYELLOW RCOLOR(ALPHA_BG,     0xff, 0xff, 0x00, 0x00, 0x00, 0x00)
-#define RColor_BWHITE    RCOLOR(ALPHA_BOLD,   0xff, 0xff, 0xff, 0x00, 0x00, 0x00)
-#define RColor_BGREEN    RCOLOR(ALPHA_BOLD,   0x00, 0xff, 0x00, 0x00, 0x00, 0x00)
-#define RColor_BMAGENTA  RCOLOR(ALPHA_BOLD,   0xff, 0x00, 0xff, 0x00, 0x00, 0x00)
-#define RColor_BYELLOW   RCOLOR(ALPHA_BOLD,   0xff, 0xff, 0x00, 0x00, 0x00, 0x00)
-#define RColor_BCYAN     RCOLOR(ALPHA_BOLD,   0x00, 0xff, 0xff, 0x00, 0x00, 0x00)
-#define RColor_BBLUE     RCOLOR(ALPHA_BOLD,   0x00, 0x00, 0xff, 0x00, 0x00, 0x00)
-#define RColor_BGRAY     RCOLOR(ALPHA_BOLD,   0x7f, 0x7f, 0x7f, 0x00, 0x00, 0x00)
+#define RColor_NULL      RCOLOR(0      , 0x00,     0x00, 0x00, 0x00, 0x00, 0x00, 0x00)
+#define RColor_BLACK     RCOLOR(0      , ALPHA_FG, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00)
+#define RColor_BGBLACK   RCOLOR(0      , ALPHA_BG, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00)
+#define RColor_RED       RCOLOR(0      , ALPHA_FG, 0xff, 0x00, 0x00, 0x00, 0x00, 0x00)
+#define RColor_BGRED     RCOLOR(0      , ALPHA_BG, 0xff, 0x00, 0x00, 0x00, 0x00, 0x00)
+#define RColor_WHITE     RCOLOR(0      , ALPHA_FG, 0xff, 0xff, 0xff, 0x00, 0x00, 0x00)
+#define RColor_BGWHITE   RCOLOR(0      , ALPHA_BG, 0xff, 0xff, 0xff, 0x00, 0x00, 0x00)
+#define RColor_GREEN     RCOLOR(0      , ALPHA_FG, 0x00, 0xff, 0x00, 0x00, 0x00, 0x00)
+#define RColor_BGGREEN   RCOLOR(0      , ALPHA_BG, 0x00, 0xff, 0x00, 0x00, 0x00, 0x00)
+#define RColor_MAGENTA   RCOLOR(0      , ALPHA_FG, 0xff, 0x00, 0xff, 0x00, 0x00, 0x00)
+#define RColor_BGMAGENTA RCOLOR(0      , ALPHA_BG, 0xff, 0x00, 0xff, 0x00, 0x00, 0x00)
+#define RColor_YELLOW    RCOLOR(0      , ALPHA_FG, 0xff, 0xff, 0x00, 0x00, 0x00, 0x00)
+#define RColor_BGYELLOW  RCOLOR(0      , ALPHA_BG, 0xff, 0xff, 0x00, 0x00, 0x00, 0x00)
+#define RColor_CYAN      RCOLOR(0      , ALPHA_FG, 0x00, 0xff, 0xff, 0x00, 0x00, 0x00)
+#define RColor_BGCYAN    RCOLOR(0      , ALPHA_BG, 0x00, 0xff, 0xff, 0x00, 0x00, 0x00)
+#define RColor_BLUE      RCOLOR(0      , ALPHA_FG, 0x00, 0x00, 0xff, 0x00, 0x00, 0x00)
+#define RColor_BGBLUE    RCOLOR(0      , ALPHA_BG, 0x00, 0x00, 0xff, 0x00, 0x00, 0x00)
+#define RColor_GRAY      RCOLOR(0      , ALPHA_FG, 0x7f, 0x7f, 0x7f, 0x00, 0x00, 0x00)
+#define RColor_BGGRAY    RCOLOR(0      , ALPHA_BG, 0x7f, 0x7f, 0x7f, 0x00, 0x00, 0x00)
+#define RColor_BBLACK    RCOLOR(1u << 1, ALPHA_FG, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00)
+#define RColor_BRED      RCOLOR(1u << 1, ALPHA_FG, 0xff, 0x00, 0x00, 0x00, 0x00, 0x00)
```

Comment:
Keep or delete, but if you keep it use it: https://github.com/radare/radare2/pull/9535/files#diff-956422fb542ea7cb490dda6e154b37dbR130

Reply:
Removed a few bold macros